regexes/device/mobiles.yml

```diff
@@ -385,8 +388,8 @@ Acer:
       model: 'Liquid Z$1'
     - regex: 'Android.*Z160 Build'
       model: 'Liquid Z4'
-    - regex: 'Android.*Z500 Build'
-      model: 'Liquid Z500'
+    - regex: 'Android.*(Z5\d{2}) Build'
+      model: 'Liquid $1'
```

Comment:
I'd suggest to have it like this:
```
- regex: 'Android.*Z(5\d{2}) Build'
  model: 'Liquid Z$1'
```
That would prove the `Z` is always a capital letter, even if in the useragent it is lower case